include/Ideen_und_Konzepte.tex

```diff
@@ -102,5 +105,17 @@ \subsubsection{Software}
 \end{figure}
  
 \subsection{Erkenntnisse der Technischen Möglichkeiten}
+Es konnte festgestellt werden, dass die Reichweite der Antenne den Herstellerangaben entsprechen, sofern der Tag parallel zur Antenne ausgerichtet ist. Ist der Tag in irgend eine Achse gedreht so funktioniert die Auslesung ab einem Auslenkungswinkel von 60\SIUnitSymbolDegree\ nicht mehr. Weiter konnte die Lesegeschwindigkeit für 15 TagIDs ermittelt werden, welche bei rund 350ms liegt. Es wurde daher geschlussfolgert, dass es machbar ist 40 TagIDs pro Sekunde zu lesen.
+Weiter konnte die seitliche Reichweite, bei einer optimalen Ausrichtung, von 25cm bei 50cm und 12.5cm bei 60cm ermittelt werden. Wie schon beschrieben können maximal ausgelenkte Tags (90\SIUnitSymbolDegree) in der Flucht nicht gelesen werden. Es wurde jedoch in den Versuchen ermittelt, dass am Rand der Antenne 30cm nach vorne diese Tags dennoch gelesen werden können (wie in Abbildung \ref{fig:Seitlich90} zu sehen).
+
+\begin{figure}[htb]
+	\centering
+	\includegraphics[keepaspectratio,height=7cm]{ResultateModell90}
+	\caption{Ein Modell der Erreichbarkeit eines Tags mit 90\SIUnitSymbolDegree Auslenkung}
+	\label{fig:Seitlich90}
+\end{figure}
+
+Weiter konnte in den Versuchen herausgefunden werden, dass ein Lesen durch ein Metall, sei dies Aluminium oder Stahl, unmöglich ist. Keiner der Tags hinter einer Metallplatte, im Minimum 0.75mm dick, konnte gelesen werden. Es konnte jedoch keine Abschirmung durch Bücher, Plastikbehälter, WLAN oder menschliche Hände festgestellt werden. Die Interferenz der Antennen konnte insofern nicht getestet werden, dass die Antennen nicht parallel betrieben werden konnten, ergo nie zwei Antennen zur selben Zeit aktiv waren. Zur Interferenz zwischen gestapelten Tags konnte eruiert werden, dass ein Mindestabstand von 1.5cm in horizontaler und 3cm in vertikaler Richtung eingehalten werden muss, dass alle Tags erfolgreich ausgelesen werden konnten.
```

Comment:
~, ergo nie zwei Antennen zur selben Zeit aktiv waren~.

Reply:
Bei gestapelten Tags konnte eine starke Interferenz festgestellt werden. Es konnte ein Mindestabstand von 1.5cm in horizontaler und 3cm in vertikaler Richtung ermittelt werden, welcher eingehalten werden muss, um alle Tags erfolgreich auslesen zu können.

Reply:
Um alle Tags erfolgreich auslesen zu können, muss ein Mindestabsond von 1.5cm in horizontaler und 3cm in vertikaler Richtung eingehalten werden.